pkg/operation/botanist/waiter.go

```diff
@@ -158,6 +156,13 @@ func (b *Botanist) WaitUntilKubeAPIServerReady(ctx context.Context) error {
 	})
 }
 
+// WaitForKubeControllerManagerToBeActive waits for the kube controller manager of a Shoot cluster has acquired leader election, thus is active.
+func (b *Botanist) WaitForKubeControllerManagerToBeActive(ctx context.Context) error {
```

Comment:
How is this change related to the rest of the changes. I think it should at least be a different commit to the helm chart refactorings

Reply:
see comment above, will do on next refactoring. Thanks